src/wallet/wallet.cpp

```diff
@@ -1940,6 +1979,7 @@ CAmount CWalletTx::GetImmatureCredit(bool fUseCache) const
 
 CAmount CWalletTx::GetAvailableCredit(bool fUseCache, const isminefilter& filter) const
 {
+    bool allow_dirty_addresses = !gArgs.GetBoolArg("-avoidreuse", DEFAULT_AVOIDREUSE);
```

Comment:
Commit "wallet: enable avoidreuse feature"

Could add a `g_allow_dirty_addresses` instead?

Reply:
Might be worth it, yeah.